keywords/LiteServNetMsft.py

```diff
@@ -52,6 +52,12 @@ def download(self):
         2. Extracts the package and removes the zip
         """
 
+        # Skip download if packages is already downloaded
+        expected_binary = "couchbase-lite-net-msft-{}-liteserv/net45/LiteServ.exe".format(self.version_build)
+        if os.path.isfile(expected_binary):
+            log_info("Package already downloaded: {}".format(expected_binary))
+            return
+
```

Comment:
I'm not sure this would work. Basically, it needs to check via the ansible script if the directory exists on the remote machine.